content/en/docs/tasks/configure-pod-container/enforce-standards-namespace-labels.md

```diff
@@ -4,7 +4,7 @@ reviewers:
 - tallclair
 - liggitt
 content_type: task
-min-kubernetes-server-version: v1.22
+min-kubernetes-server-version: v1.25
```

Comment:
Same question here

Reply:
Pod Security Admission is GA in Kubernetes 1.25

Reply:
I've revised this; a preview should already be available.